trezor-common/src/tests/mod.rs

```diff
@@ -146,6 +146,10 @@ impl From<chain::TxInput> for crate::TxInput {
             chain::TxInput::AccountCommand(nonce, command) => {
                 Self::AccountCommand(nonce.value(), command.into())
             }
+            chain::TxInput::OrderAccountCommand(_) => {
+                //TODO: support OrdersVersion::V1
```

Comment:
I guess this should be put into https://github.com/mintlayer/mintlayer-core/issues/1887

Or maybe some other issue

Reply:
https://github.com/mintlayer/mintlayer-core/issues/1902